onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -2510,13 +2511,13 @@ TensorrtExecutionProvider::GetCapability(const GraphViewer& graph,
 /**
  * Refit the weight-stripped engine
  */
-common::Status TensorrtExecutionProvider::RefitEngine(std::string onnx_model_filename,
-                                                      std::string& onnx_model_folder_path,
-                                                      std::string& weight_stripped_engine_cath_path,
-                                                      bool path_check,
-                                                      nvinfer1::ICudaEngine* trt_engine,
-                                                      bool serialize_refitted_engine,
-                                                      bool detailed_build_log) {
+common::Status TensorrtExecutionProvider::RefitEngine([[maybe_unused]] std::string onnx_model_filename,
```

Comment:
the function is divided into two ifdef branches. consider putting ORT_UNUSED_PARAMETERs in the 'not implemented' branch so we can still get warnings in the other branch.

Reply:
Reverted the change to keep it unchanged.